### PR TITLE
Merchant dashboard

### PR DESCRIPTION
**Pull request by Freeing3092**
Add merchant_top_5_customers method and view.
Add Merchant Dashboard Invoice Dates and sorting

### CODE REVIEW

File: spec/features/merchants/show_spec.rb [hunk 1]
```diff
@@ -4,71 +4,132 @@
...
+    @customer3 = create(:customer)
+    @customer1_invoice = create(:invoice, customer: @customer1)
+    @customer2_invoice = create(:invoice, customer: @customer2)
+    @customer3_invoice = create(:invoice, customer: @customer3)
+    
+    @customer1_invoice_item = create(:invoice_item, invoice: @customer1_invoice, item: @item_1, status: 0)
+    @customer2_invoice_item = create(:invoice_item, invoice: @customer2_invoice, item: @item_2, status: 1)
+    @customer3_invoice_item = create(:invoice_item, invoice: @customer3_invoice, item: @item_3, status: 2)
+    
+    visit "/merchants/#{@merchant.id}/dashboard"
```
Comment (gabrielcnunez) [inline spec/features/merchants/show_spec.rb:22]: Did not know you could put a URL in the before each block, very cool!

**Review by gabrielcnunez — CHANGES_REQUESTED** [review]
Can you delete the .DS_Store file from your local repo, commit and push again por favor?

**Comment by Freeing3092** [comment]
.DS_Store removed and resolved merge conflicts.

**Review by gabrielcnunez — APPROVED** [review]
Appreciate the update, thanks Rich!

**Pull request merged**